docs/it/user/configuration.rst

```diff
@@ -47,9 +47,11 @@ Potete cambiare il vostro nome, il vostro indirizzo email e abilitare l'``Autent
 Autenticazione a due fattori (2FA)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-L'autenticazione a due fattori (conosciuta anche come 2FA) é una tecnologia brevettata nel 1984 che offre l'identificazione degli utenti tramite una combinazione di due componenti differenti.
+    L'autenticazione a due fattori (conosciuta anche come 2FA) é una tecnologia brevettata nel 1984 che offre l'identificazione degli    utenti tramite una combinazione di due componenti differenti.
```

Comment:
Ok no problem, I saw them in the English version too, so I though it was correct to insert them.